css/includes/components/_asset-form.scss

```diff
@@ -61,4 +61,8 @@
          }
       }
    }
+
+   .form-field-ajax > .mb-3 {
```

Comment:
I'm not very fan of this override, could you pass a specific class into the html loaded by ajax, or better remove `mb-3` (maybe adding a flag) in case of ajax load